pkg/kdk/config.go

```diff
@@ -0,0 +1,123 @@
+// Copyright © 2018 Cisco Systems, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kdk
+
+import (
+	"context"
+	"os/user"
+	"path/filepath"
+	"strings"
+
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/client"
+	"github.com/mitchellh/go-homedir"
+)
+
+// Struct of all configs to be saved directly as ~/.kdk/<NAME>/config.yaml
+type KdkEnvConfig struct {
```

Comment:
`KdkEnvConfig` is kind of redundant, since this is the `kdk` package.  Is there a way to simplify this name so as to follow go naming idioms?  

(definitely a `May`)